mmv1/templates/terraform/examples/datastream_stream_oracle.tf.erb

```diff
@@ -0,0 +1,86 @@
+resource "google_datastream_connection_profile" "source" {
```

Comment:
Both examples start off indented, can we shift left?

Reply:
not sure I understand, for me they seems to start with indent 0

Reply:
error on my end, apologies